ui/pages/confirmations/confirmation/templates/index.js

```diff
@@ -46,6 +48,7 @@ const APPROVAL_TEMPLATES = {
   [ApprovalType.SnapDialogAlert]: snapAlert,
   [ApprovalType.SnapDialogConfirmation]: snapConfirmation,
   [ApprovalType.SnapDialogPrompt]: snapPrompt,
+  [DIALOG_APPROVAL_TYPES.default]: snapDefault,
```

Comment:
I guess we should upstream this to `ApprovalType`

Reply:
Yup, this is just a temporary solution

Reply:
Please create a ticket or a PR so we don't forget to do this